### PR TITLE
add solv-funds

### PR DESCRIPTION
**Pull request by buchaoqun**
All open fund pools.

### CODE REVIEW

**Comment by llamatester** [comment]
The solv-funds adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       82 passed, 82 total
    Snapshots:   0 total
    Time:        0.304 s
    Ran all test suites.
    
    Nb of pools: 13
     
    
    Sample pools:
    ┌─────────┬───────────────────────────────────────────────────────────────────────────────┬────────────┬──────────────┬────────┬──────────────────────────────────────────────────┬────────────────────┬────────┬─────────────────────────────────────────────────────┐
    │ (index) │                                     pool                                      │   chain    │   project    │ symbol │                 underlyingTokens                 │       tvlUsd       │  apy   │                         url                         │
    ├─────────┼───────────────────────────────────────────────────────────────────────────────┼────────────┼──────────────┼────────┼──────────────────────────────────────────────────┼────────────────────┼────────┼─────────────────────────────────────────────────────┤
    │    0    │ '0x392c65b44c03fee27826b645ebc00f3cfd114dc7086523523233766ce0cbd4cf-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │ 13107938.25681648  │ 12.18  │ 'https://app.solv.finance/earn/open-fund/detail/30' │
    │    1    │ '0xc71ee4ace4d770871f1ead92c5fb5e39ea888e8660a8bd27ed3f9423140e6576-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'WETH' │ [ '0x82af49447d8a07e3bd95bd0d56f35241523fbab1' ] │ 5107397.080070815  │ 10.85  │ 'https://app.solv.finance/earn/open-fund/detail/31' │
    │    2    │ '0xe2d154c8cd3f45e2a22572fad8301b195755cf9af10d579e04e90f719206fad0-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'WBTC' │ [ '0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f' ] │ 3783101.8149285004 │ 12.24  │ 'https://app.solv.finance/earn/open-fund/detail/32' │
    │    3    │ '0xe037ef7b5f74bf3c988d8ae8ab06ad34643749ba9d217092297241420d600fce-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │ 3659454.9435002445 │ 22.14  │ 'https://app.solv.finance/earn/open-fund/detail/5'  │
    │    4    │ '0x504d291d2f4dedf8fa3ac3a342ff3531b8947fa835077c8312fa18da2be4084c-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │      2007784       │  5.76  │ 'https://app.solv.finance/earn/open-fund/detail/33' │
    │    5    │ '0x9119ceb6bcf974578e868ab65ae20c0d546716a6657eb27dc3a6bf113f0b519c-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDT' │ [ '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9' ] │ 1004853.8499999999 │  4.9   │ 'https://app.solv.finance/earn/open-fund/detail/22' │
    │    6    │  '0xf514bf1cefa177ea26156b1d0d0da28da8f88b5eb8ed673ddd2cea5334bb6d6c-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │ 402359.80129392963 │  4.4   │ 'https://app.solv.finance/earn/open-fund/detail/10' │
    │    7    │ '0x3b2232fb5309e89e5ee6e2ca6066bcc28ee365045e9a565040bf8c846b87477e-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'WBTC' │ [ '0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f' ] │  300923.00708742   │   0    │ 'https://app.solv.finance/earn/open-fund/detail/40' │
    │    8    │ '0x375ebcd78e8b3571c0f6482bdaae602672e73e145e92ca40f9b8f1537236bf2e-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDT' │ [ '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9' ] │ 180965.0520240594  │ -95.74 │ 'https://app.solv.finance/earn/open-fund/detail/1'  │
    │    9    │  '0x370c1ec55ca4a79b54909892f55573603611e0cf34f89a40405e81b8d6a78195-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │ 157423.09449398366 │  4.75  │ 'https://app.solv.finance/earn/open-fund/detail/38' │
    └─────────┴───────────────────────────────────────────────────────────────────────────────┴────────────┴──────────────┴────────┴──────────────────────────────────────────────────┴────────────────────┴────────┴─────────────────────────────────────────────────────┘
    This adapter contains some pools with <10k TVL, these pools won't be shown in DefiLlama

**Review by christianlantin — APPROVED** [review]

File: src/adaptors/solv-funds/index.js [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  }
+
+  return ustPool; // Anchor only has a single pool with APY
+};
+
+module.exports = {
+  timetravel: false,
+  apy: poolsFunction,
+  url: 'https://app.solv.finance/',
+};
```
Comment (christianlantin) [inline src/adaptors/solv-funds/index.js:77]: src/adaptors/solv-funds/index.js

**Comment by slasher125** [comment]
hi, ty for adding

- the institution only pools are not really useful in our context, given most won't be able to participate. lets remove these
- apy needs to be after fees
- ideally split it into apyBase and apyReward (if required) where apyBase = anything from fees, apyReward = incentives
- are there any lockups in these strategies? if so, needs to be mentioned under `poolMeta`

**Comment by llamatester** [comment]
The solv-funds adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       82 passed, 82 total
    Snapshots:   0 total
    Time:        0.316 s
    Ran all test suites.
    
    Nb of pools: 13
     
    
    Sample pools:
    ┌─────────┬───────────────────────────────────────────────────────────────────────────────┬────────────┬──────────────┬────────┬──────────────────────────────────────────────────┬────────────────────┬────────┬─────────────────────────────────────────────────────┐
    │ (index) │                                     pool                                      │   chain    │   project    │ symbol │                 underlyingTokens                 │       tvlUsd       │  apy   │                         url                         │
    ├─────────┼───────────────────────────────────────────────────────────────────────────────┼────────────┼──────────────┼────────┼──────────────────────────────────────────────────┼────────────────────┼────────┼─────────────────────────────────────────────────────┤
    │    0    │ '0x392c65b44c03fee27826b645ebc00f3cfd114dc7086523523233766ce0cbd4cf-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │ 13105203.208079075 │ 11.82  │ 'https://app.solv.finance/earn/open-fund/detail/30' │
    │    1    │ '0xc71ee4ace4d770871f1ead92c5fb5e39ea888e8660a8bd27ed3f9423140e6576-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'WETH' │ [ '0x82af49447d8a07e3bd95bd0d56f35241523fbab1' ] │ 4994972.568920889  │ 10.41  │ 'https://app.solv.finance/earn/open-fund/detail/31' │
    │    2    │ '0xe037ef7b5f74bf3c988d8ae8ab06ad34643749ba9d217092297241420d600fce-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │ 3664460.5042338087 │ 20.66  │ 'https://app.solv.finance/earn/open-fund/detail/5'  │
    │    3    │ '0xe2d154c8cd3f45e2a22572fad8301b195755cf9af10d579e04e90f719206fad0-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'WBTC' │ [ '0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f' ] │ 3660604.651626472  │ 11.74  │ 'https://app.solv.finance/earn/open-fund/detail/32' │
    │    4    │ '0x504d291d2f4dedf8fa3ac3a342ff3531b8947fa835077c8312fa18da2be4084c-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │ 2007537.6886500001 │  5.77  │ 'https://app.solv.finance/earn/open-fund/detail/33' │
    │    5    │ '0x9119ceb6bcf974578e868ab65ae20c0d546716a6657eb27dc3a6bf113f0b519c-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDT' │ [ '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9' ] │      1003980       │  4.89  │ 'https://app.solv.finance/earn/open-fund/detail/22' │
    │    6    │  '0xf514bf1cefa177ea26156b1d0d0da28da8f88b5eb8ed673ddd2cea5334bb6d6c-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │ 401957.8434504792  │  4.4   │ 'https://app.solv.finance/earn/open-fund/detail/10' │
    │    7    │ '0x3b2232fb5309e89e5ee6e2ca6066bcc28ee365045e9a565040bf8c846b87477e-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'WBTC' │ [ '0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f' ] │  291175.99369926   │   0    │ 'https://app.solv.finance/earn/open-fund/detail/40' │
    │    8    │ '0x375ebcd78e8b3571c0f6482bdaae602672e73e145e92ca40f9b8f1537236bf2e-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDT' │ [ '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9' ] │ 188584.5357082949  │ -93.73 │ 'https://app.solv.finance/earn/open-fund/detail/1'  │
    │    9    │  '0x370c1ec55ca4a79b54909892f55573603611e0cf34f89a40405e81b8d6a78195-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │ 168558.27222987625 │ -0.14  │ 'https://app.solv.finance/earn/open-fund/detail/38' │
    └─────────┴───────────────────────────────────────────────────────────────────────────────┴────────────┴──────────────┴────────┴──────────────────────────────────────────────────┴────────────────────┴────────┴─────────────────────────────────────────────────────┘
    This adapter contains some pools with <10k TVL, these pools won't be shown in DefiLlama

**Comment by llamatester** [comment]
The solv-funds adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       53 passed, 53 total
    Snapshots:   0 total
    Time:        0.315 s
    Ran all test suites.
    
    Nb of pools: 7
     
    
    Sample pools:
    ┌─────────┬───────────────────────────────────────────────────────────────────────────────┬────────────┬──────────────┬────────┬──────────────────────────────────────────────────┬────────────────────┬─────────┬───────────┬──────────────────────────────────────────────────┬─────────────────────────────────────────────────────┐
    │ (index) │                                     pool                                      │   chain    │   project    │ symbol │                 underlyingTokens                 │       tvlUsd       │ apyBase │ apyReward │                   rewardTokens                   │                         url                         │
    ├─────────┼───────────────────────────────────────────────────────────────────────────────┼────────────┼──────────────┼────────┼──────────────────────────────────────────────────┼────────────────────┼─────────┼───────────┼──────────────────────────────────────────────────┼─────────────────────────────────────────────────────┤
    │    0    │ '0xe037ef7b5f74bf3c988d8ae8ab06ad34643749ba9d217092297241420d600fce-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │ 3665505.173208173  │  20.16  │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/5'  │
    │    1    │ '0x504d291d2f4dedf8fa3ac3a342ff3531b8947fa835077c8312fa18da2be4084c-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │      2008110       │  5.27   │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/33' │
    │    2    │ '0x9119ceb6bcf974578e868ab65ae20c0d546716a6657eb27dc3a6bf113f0b519c-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDT' │ [ '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9' ] │ 1004983.9799999999 │  4.39   │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/22' │
    │    3    │  '0xf514bf1cefa177ea26156b1d0d0da28da8f88b5eb8ed673ddd2cea5334bb6d6c-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │ 402359.80129392963 │   4.4   │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/10' │
    │    4    │  '0x370c1ec55ca4a79b54909892f55573603611e0cf34f89a40405e81b8d6a78195-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │ 168726.8305021061  │  -0.14  │  129.65   │ [ '0x779f4e5fb773e17bc8e809f4ef1abb140861159a' ] │ 'https://app.solv.finance/earn/open-fund/detail/38' │
    │    5    │  '0x3f9c68bbb64152799c006c8c505d60edf930f7a45db3b63534ed4c2dae07f7a8-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │ 100099.99999999999 │    0    │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/34' │
    │    6    │ '0xc573ea3cb55168e5ebe04b118c4b8a33ba62e86bbe1d1e735a447467795c3bd4-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │ 21208.976589592967 │  9.77   │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/2'  │
    └─────────┴───────────────────────────────────────────────────────────────────────────────┴────────────┴──────────────┴────────┴──────────────────────────────────────────────────┴────────────────────┴─────────┴───────────┴──────────────────────────────────────────────────┴─────────────────────────────────────────────────────┘

**Comment by buchaoqun** [comment]
Made adjustments

**Comment by slasher125** [comment]
ty, another smol thing:
- pls make use of poolMeta to add some additional context eg `GMX V2 Senior Tranche` to help distinguish btw pools

**Comment by llamatester** [comment]
The solv-funds adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       53 passed, 53 total
    Snapshots:   0 total
    Time:        0.308 s
    Ran all test suites.
    
    Nb of pools: 7
     
    
    Sample pools:
    ┌─────────┬───────────────────────────────────────────────────────────────────────────────┬────────────┬──────────────┬────────┬──────────────────────────────────────────────────┬────────────────────┬─────────┬───────────┬──────────────────────────────────────────────────┬─────────────────────────────────────────────────────┐
    │ (index) │                                     pool                                      │   chain    │   project    │ symbol │                 underlyingTokens                 │       tvlUsd       │ apyBase │ apyReward │                   rewardTokens                   │                         url                         │
    ├─────────┼───────────────────────────────────────────────────────────────────────────────┼────────────┼──────────────┼────────┼──────────────────────────────────────────────────┼────────────────────┼─────────┼───────────┼──────────────────────────────────────────────────┼─────────────────────────────────────────────────────┤
    │    0    │ '0xe037ef7b5f74bf3c988d8ae8ab06ad34643749ba9d217092297241420d600fce-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │ 3665505.173208173  │  20.16  │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/5'  │
    │    1    │ '0x504d291d2f4dedf8fa3ac3a342ff3531b8947fa835077c8312fa18da2be4084c-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │      2008434       │  5.51   │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/33' │
    │    2    │ '0x9119ceb6bcf974578e868ab65ae20c0d546716a6657eb27dc3a6bf113f0b519c-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDT' │ [ '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9' ] │      1004110       │  4.56   │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/22' │
    │    3    │  '0xf514bf1cefa177ea26156b1d0d0da28da8f88b5eb8ed673ddd2cea5334bb6d6c-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │ 401957.8434504792  │   4.4   │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/10' │
    │    4    │  '0x370c1ec55ca4a79b54909892f55573603611e0cf34f89a40405e81b8d6a78195-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │ 168558.27222987625 │  -0.14  │  129.75   │ [ '0x779f4e5fb773e17bc8e809f4ef1abb140861159a' ] │ 'https://app.solv.finance/earn/open-fund/detail/38' │
    │    5    │  '0x3f9c68bbb64152799c006c8c505d60edf930f7a45db3b63534ed4c2dae07f7a8-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │       100000       │    0    │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/34' │
    │    6    │ '0xc573ea3cb55168e5ebe04b118c4b8a33ba62e86bbe1d1e735a447467795c3bd4-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │ 21208.976589592967 │  9.77   │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/2'  │
    └─────────┴───────────────────────────────────────────────────────────────────────────────┴────────────┴──────────────┴────────┴──────────────────────────────────────────────────┴────────────────────┴─────────┴───────────┴──────────────────────────────────────────────────┴─────────────────────────────────────────────────────┘

**Comment by llamatester** [comment]
The solv-funds adapter exports pools: 
        
 
 Test Suites: 1 passed, 1 total
    Tests:       53 passed, 53 total
    Snapshots:   0 total
    Time:        0.308 s
    Ran all test suites.
    
    Nb of pools: 7
     
    
    Sample pools:
    ┌─────────┬───────────────────────────────────────────────────────────────────────────────┬────────────┬──────────────┬────────┬──────────────────────────────────────────────────┬────────────────────┬─────────┬───────────┬──────────────────────────────────────────────────┬─────────────────────────────────────────────────────┬─────────────────────────────────────────────────────┐
    │ (index) │                                     pool                                      │   chain    │   project    │ symbol │                 underlyingTokens                 │       tvlUsd       │ apyBase │ apyReward │                   rewardTokens                   │                         url                         │                      poolMeta                       │
    ├─────────┼───────────────────────────────────────────────────────────────────────────────┼────────────┼──────────────┼────────┼──────────────────────────────────────────────────┼────────────────────┼─────────┼───────────┼──────────────────────────────────────────────────┼─────────────────────────────────────────────────────┼─────────────────────────────────────────────────────┤
    │    0    │ '0xe037ef7b5f74bf3c988d8ae8ab06ad34643749ba9d217092297241420d600fce-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │ 3665505.173208173  │  20.16  │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/5'  │             'GMX V2 Delta Neutral Pool'             │
    │    1    │ '0x504d291d2f4dedf8fa3ac3a342ff3531b8947fa835077c8312fa18da2be4084c-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │      2008434       │  5.51   │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/33' │               'GMX V2 Senior Tranche'               │
    │    2    │ '0x9119ceb6bcf974578e868ab65ae20c0d546716a6657eb27dc3a6bf113f0b519c-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDT' │ [ '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9' ] │      1004110       │  4.56   │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/22' │  'RWA: Generate Yield On Your Stable Coins - ARB'   │
    │    3    │  '0xf514bf1cefa177ea26156b1d0d0da28da8f88b5eb8ed673ddd2cea5334bb6d6c-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │ 401957.8434504792  │   4.4   │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/10' │     'RWA: Generate Yield On Your Stable Coins'      │
    │    4    │  '0x370c1ec55ca4a79b54909892f55573603611e0cf34f89a40405e81b8d6a78195-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │ 168558.27222987625 │  -0.14  │  129.75   │ [ '0x779f4e5fb773e17bc8e809f4ef1abb140861159a' ] │ 'https://app.solv.finance/earn/open-fund/detail/38' │              'KTX Delta Neutral Pool'               │
    │    5    │  '0x3f9c68bbb64152799c006c8c505d60edf930f7a45db3b63534ed4c2dae07f7a8-mantle'  │  'mantle'  │ 'solv-funds' │ 'USDT' │ [ '0x201eba5cc46d216ce6dc03f6a759e8e766e956ae' ] │       100000       │    0    │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/34' │ 'RWA: Generate Yield On Your Stable Coins - Mantle' │
    │    6    │ '0xc573ea3cb55168e5ebe04b118c4b8a33ba62e86bbe1d1e735a447467795c3bd4-arbitrum' │ 'arbitrum' │ 'solv-funds' │ 'USDC' │ [ '0xaf88d065e77c8cc2239327c5edb3a432268e5831' ] │ 21208.976589592967 │  9.77   │     0     │                        []                        │ 'https://app.solv.finance/earn/open-fund/detail/2'  │             'GMX V1 Delta Neutral Pool'             │
    └─────────┴───────────────────────────────────────────────────────────────────────────────┴────────────┴──────────────┴────────┴──────────────────────────────────────────────────┴────────────────────┴─────────┴───────────┴──────────────────────────────────────────────────┴─────────────────────────────────────────────────────┴─────────────────────────────────────────────────────┘

**Comment by buchaoqun** [comment]
Thank you, it has been adjusted.

**Pull request merged**